### PR TITLE
Add Default Mobile Styling to Vanguard 2019 Article Series

### PR DESCRIPTION
**Pull request by ashleyjelks**
This PR adds support for mobile styling when mobile browsers do not support mix blend modes.

[Links to GROW-1509](https://artsyproduct.atlassian.net/browse/GROW-1509)

After:

Kapture 2019-09-09 at 14 28 34

Before:

Screenshot_20190909-143255
Screenshot_20190909-143241

### CODE REVIEW

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx [hunk 1]
```diff
@@ -108,7 +113,14 @@ export class VanguardArtistWrapper extends React.Component<
         >
           <BackgroundContainer>{background}</BackgroundContainer>
           <ArticleWithFullScreen article={article}>
-            <ArtistContainer pb={4} maxWidth={["100vw", 1000]} px={4} mx="auto">
+            <ArtistContainer
+              pb={4}
+              maxWidth={["100vw", 1000]}
+              px={4}
+              mx="auto"
+              isMobile
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: I would expect this to use a prop, it looks like we are setting the `ArtistContainer` `isMobile` to `true` for all instances.

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx [hunk 2]
```diff
@@ -92,10 +94,6 @@ const SubSeriesHeaderText = styled(Box)`
   left: 0;
   top: 50px;
   right: 0;
-  mix-blend-mode: difference;
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx]: Is it intentional to remove the blend mode from the header text? We should do this only for mobile.

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx [hunk 3]
```diff
@@ -117,14 +129,24 @@ export class VanguardArtistWrapper extends React.Component<
...
                   </InvertedSerif>
                 </Media>
 
                 <Box position="relative">
                   {hero_section && (
-                    <InvertedSans size="4" weight="medium">
+                    <InvertedSans
+                      size="4"
+                      weight="medium"
+                      isMobile
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: maybe should be a prop

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx [hunk 4]
```diff
@@ -153,6 +175,8 @@ export class VanguardArtistWrapper extends React.Component<
                 weight="medium"
                 textAlign="center"
                 onClick={this.onExpand.bind(this)}
+                isMobile
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: maybe should be a prop

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx (hunk 1 above)
Comment (ashleyjelks) [reply src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: Ah, ok yeah at first glance it does look like its always `true` but `isMobile` is a boolean prop passed here from the `SeriesWrapper` component:

```
{relatedArticles &&
        relatedArticles.map((artistArticle, i) => (
          <VanguardArtistWrapper
            key={i}
            article={artistArticle}
            section={slugifiedTitle}
            isMobile={isMobile}
          />
        ))}
```

Comment (eessex) [reply src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx]: I would still expect this to use `isMobile={isMobile}` if we are passing a value, the declaration in the code should mean always true.

**Comment by artsyit** [comment]
:rocket: PR was released in v17.11.4 :rocket:

**Pull request merged**